lib/chef/knife/winrm_shared_options.rb

```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
       # Shared command line options for knife winrm and knife wsman test
       def self.included(includer)
         includer.class_eval do
-          option :manual,
+          option :manual_list,
             :short => "-m",
             :long => "--manual-list",
             :boolean => true,
             :description => "QUERY is a space separated list of servers",
             :default => false
 
-          option :attribute,
-            :short => "-a ATTR",
-            :long => "--attribute ATTR",
+          option :winrm_attribute,
```

Comment:
I'm just saying that the `:option`, short and long arg names on the cli need to be the same in `knife winrm` and `knife ssh`.